server/sonar-web/src/main/js/helpers/testUtils.js

```diff
@@ -21,7 +21,8 @@ export const click = element => {
   return element.simulate('click', {
     target: { blur() {} },
     currentTarget: { blur() {} },
-    preventDefault() {}
+    preventDefault() {},
+    stopPropagation() {}
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this function is empty or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS1186)